lib/internal/streams/destroy.js

```diff
@@ -7,6 +7,15 @@ function destroy(err, cb) {
   const r = this._readableState;
   const w = this._writableState;
 
+  if ((w && w.destroyed) || (r && r.destroyed)) {
+    if (typeof cb === 'function') {
+      // TODO(ronag): Invoke with `'close'`/`'error'`.
```

Comment:
Does it make sense to have this fixed in the same PR or make is separate?

Reply:
I think it's better to use a dedicated PR.